packages/server/tests/acceptance/htsPrecompile.spec.ts

```diff
@@ -0,0 +1,95 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// external resources
+import { solidity } from 'ethereum-waffle';
+import chai, { expect } from 'chai';
+
+chai.use(solidity);
+
+import { AliasAccount } from '../clients/servicesClient';
+import { ethers, BigNumber } from 'ethers';
+import ERC20MockJson from '../contracts/ERC20Mock.json';
+import BaseHTSJson from '../contracts/BaseHTS.json';
+
+
+describe('HTS Precompile Acceptance Tests', async function() {
+  this.timeout(240 * 1000); // 240 seconds
+  const { servicesNode, relay } = global;
+
+  const accounts: AliasAccount[] = [];
+  let baseHTSContract;
+  let HTSTokenContract;
+
+  before(async () => {
+    accounts[0] = await servicesNode.createAliasAccount(1000, relay.provider);
```

Comment:
nit: Could you lower the initial balance of the alias accounts. Somewhere in the range of 10-30 hbars.

Reply:
Yep, as low as possible to pass in remote envs like previewnet/testnet